### DISCUSSION TITLE
[REQUEST] Tailscale LXC

### DISCUSSION THREAD

**Discussion by oldcrazyeye** [tteck/Proxmox#317] [Ideas]
just wondering if its possible for you to make a tailscale lxc script or even a vm script. whichever is easiest. Thanks tteck

**Comment by tteck** [comment]
Would you please test
```

```

**Reply by tteck** [reply]
Another **test** @oldcrazyeye 
To add Tailscale to an existing LXC
```
Main branch
```
Remember to change `106` to the LXC ID you're wanting to add Tailscale
Reboot the LXC and then `tailscale up` in the LXC console

**Reply by oldcrazyeye** [reply]
that works! thanks man

**Comment by tteck** [comment]
I'm not seeing a need for a Tailscale LXC install script if we can just add Tailscale to an existing LXC.
What's everyone's thoughts?

**Reply by oldcrazyeye** [reply]
i think thats perfectly fine

**Reply by oldcrazyeye** [reply]
Would just need to add tailscale to any lxc you want access to. But with that script it's easy enough

**Reply by tteck** [reply]
If it was easy as just installing Tailscale (`curl -fsSL https://tailscale.com/install.sh | sh`), I wouldn't even write a script.
But the `/etc/pve/lxc/${CTID}.conf` has to be edited to allow Tun/Tap

**Reply by oldcrazyeye** [reply]
Hence why I requested the script haha

**Reply by kienzlenator** [reply]
Sorry for the necro here. My guess is someone is asking for a container to be preconfigured as a subnet router and exit node. Not gonna lie, that would be awesome.
